Tasks/Common/sonarqube-common/sonarqube-common.ts

```diff
@@ -39,26 +46,41 @@ export function applySonarQubeAnalysisParams(toolRunner:ToolRunner, sqProjectNam
     return toolRunner;
 }
 
-export class SonarQubeEndpoint {
-    constructor(public Url, public Username, public Password) {
+// Run SQ analysis in issues mode, but only in PR builds
+export function applySonarQubeIssuesModeInPrBuild(toolrunner: ToolRunner) {
```

Comment:
The args are identical for both Maven and Gradle so it makes sense to use a shared location for this logic.